metaseq/scripts/convert_to_singleton.py

```diff
@@ -89,10 +89,10 @@ def _build_model(cfg, task):
             for r, t in enumerate(gathered):
                 model_parts[r][name] = t.cpu()
 
-    glued = glue_megatron_parts(model_parts)
+    glued = reshard_megatron_parts(model_parts, new_model_part_count=1)[0]
     # glued['decoder.output_projection.weight'] = glued['decoder.embed_tokens.weight']
 
-    glued["decoder.version"] = model["model"]["decoder.version"].cpu()
+    glued["decoder.version"] = model.state_dict()["decoder.version"].cpu()
```

Comment:
The previous line doesn't work because of this error 

```
  File "/shared/home/punitkoura/src/metaseq/metaseq/scripts/convert_to_singleton.py", line 95, in worker_main                                                                                                                                                 
    glued["decoder.version"] = model["model"]["decoder.version"].cpu()                                                                                                                                                                                        
  File "/shared/home/punitkoura/src/fairscale/fairscale/nn/data_parallel/fully_sharded_data_parallel.py", line 889, in __getitem__                                                                                                                            
    return self.module.__getitem__(key)                                                                                                                                                                                                                       
  File "/shared/home/punitkoura/src/fairscale/fairscale/nn/misc/flatten_params_wrapper.py", line 427, in __getitem__                                                                                                                                          
    return self.module.__getitem__(key)                                                                                                                                                                                                                       
  File "/shared/home/punitkoura/miniconda3/envs/fairseq-20220503/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1185, in __getattr__    
    raise AttributeError("'{}' object has no attribute '{}'".format(                                                                                                                                                                                          
AttributeError: 'ModelParallelTransformerLanguageModel' object has no attribute '__getitem__'  
```

, therefore getting the decoder version from the state dict.